### PR TITLE
fixed code coverage publication script

### PR DESCRIPTION
**Pull request by nicflores**
afterSuccess script fixed to publish codecov data

### CODE REVIEW

File: scripts/afterSuccess [hunk 1]
```diff
@@ -8,6 +8,7 @@ source "$(dirname $0)/constants"
 # Only run for the first travis job
 if [[ "${TRAVIS_JOB_NUMBER##*.}" == "1" ]]; then
   "$SCRIPT_DIR/publishJarIfMaster"
+elif [[ "${TRAVIS_JOB_NUMBER##*.}" == "5" ]]; then
```
Comment (drostron) [inline scripts/afterSuccess]: Not introduced in this PR, but using `TRAVIS_JOB_NUMBER`s are a bit fragile. Perhaps we should switch to `CONNECTOR` if sufficient.

**Comment by drostron** [comment]
Needs a merge from master

File: scripts/afterSuccess (hunk 1 above)
Comment (nicflores) [inline scripts/afterSuccess]: Good point. Should we make that switch now?

Comment (drostron) [inline scripts/afterSuccess]: Up to you. I'll wait to merge if you want to make the change now.

**Comment by drostron** [comment]
LGTM

**Pull request merged**